android/lib/config/api_endpoints.dart

```diff
@@ -0,0 +1,4 @@
+const int serverPort = 80;
```

Comment:
These parameters can be set as environment variables for a later time so that we can configure the app for integration tests with Github Actions

Reply:
Thank you for your feedback, noted!